lib/plugsnag.ex

```diff
@@ -19,8 +21,26 @@ defmodule Plugsnag do
         Application.get_env(:plugsnag, :reporter, Bugsnag)
       end
 
-      defp handle_errors(_conn, %{reason: exception}) do
-        reporter.report(exception)
+      defp handle_errors(conn, %{reason: exception}) do
+        reporter.report(exception, metadata: extract_metadata(conn))
+      end
+
+      defp extract_metadata(conn) do
+        %{
+          conn: %{
+            request_path: conn.request_path,
+            method: conn.method,
+            port: conn.port,
+            scheme: conn.scheme,
+            query_string: conn.query_string
+          }
+        }
+      end
+
+      defp format_ip(ip) do
+        ip
+        |> Tuple.to_list
+        |> Enum.join(".")
```

Comment:
Yup. Definitely looks like an oversight on my part. I'll sort that out.

On Fri, Aug 5, 2016 at 6:45 AM Aaron Renner <notifications@github.com>
wrote:

> In lib/plugsnag.ex
> <https://github.com/jarednorman/plugsnag/pull/12#discussion_r73599282>:
>
> > +      defp extract_metadata(conn) do
> > +        %{
> > +          conn: %{
> > +            request_path: conn.request_path,
> > +            method: conn.method,
> > +            port: conn.port,
> > +            scheme: conn.scheme,
> > +            query_string: conn.query_string
> > +          }
> > +        }
> > +      end
> > +
> > +      defp format_ip(ip) do
> > +        ip
> > +        |> Tuple.to_list
> > +        |> Enum.join(".")
>
> I don't quite see where format_ip is being called.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/jarednorman/plugsnag/pull/12/files/07da97d73906eb06d4481ee4e64893ad7e968bf9#r73599282>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAHv9xuMRaVzl1NdF5R9lfO17-xYkY2Hks5qck9fgaJpZM4JXNTK>
> .
>